MathFunction.java

```diff
@@ -0,0 +1,20 @@
+package com.company.hw01;
+
+public class MathFunction {
+
+    public double getPow(float x, double pow){
+        double result = x;
+        if(pow > 0){
+            for(int i = 1; i < pow; i ++){
+                result*=x;
+            }
+            return result;
+        }else if(pow < 0){
+            for(int i = 1; i < -pow; i ++){

```

Comment:
this will work but looks awkward, extract for loop in separate method to remove code duplicates